cmd/Makefile.am

```diff
@@ -385,10 +390,6 @@ if APPARMOR
 endif
 	rmdir $(DESTDIR)$(snapdstatedir)/void || true
 
-install-exec-hook:
```

Comment:
So I realize that we do not setcap the file. So the TODO about the strip, I think it was never setcap at all.

Reply:
I wonder if we should "setcap || true". Or print only warning if it did not work.